lib/import/helpers/brca/providers/rj1/rj1_common_methods_processor.rb

```diff
@@ -0,0 +1,133 @@
+module Import
+  module Helpers
+    module Brca
+      module Providers
+        module Rj1
+          # Processor for common exctraction methods for FullScreen, Targeted, Ashkenazi and Polish
+          module Rj1CommonMethodsProcessor
```

Comment:
Perhaps not something to correct this time around, but if the module is already namespaced inside `Rj1`, the module name itself probably doesn't need to also include `Rj1`